src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -96,6 +101,14 @@ private void setSubmissionData(Map<String, Object> data, String overseasEntityId
         data.put(MANAGING_OFFICERS_INDIVIDUAL_FIELD, submissionDto.getManagingOfficersIndividual());
         data.put(MANAGING_OFFICERS_CORPORATE_FIELD, submissionDto.getManagingOfficersCorporate());
         data.put(BENEFICIAL_OWNERS_STATEMENT, submissionDto.getBeneficialOwnersStatement());
+
+        // Convert trust data to JSON string
+        ObjectMapper mapper = JsonMapper.builder().findAndAddModules().build();
```

Comment:
could you add a test to the FilingsServiceTest to test that the  submissionDto.getTrusts() get's converted to a string correctly?

Reply:
Yes. It should be checking that those NON_NULL definitions are working (and when not present, null is output).

Also need a test when no trust data is present.